packages/core/utils/src/dml/helpers/create-mikro-orm-entity.ts

```diff
@@ -170,7 +174,7 @@ export function createMikrORMEntity() {
 
       const providerEntityIdIndexStatement = createPsqlIndexStatementHelper({
         name,
-        tableName,
+        tableName: pgSchema ? `${pgSchema}"."${tableName}` : tableName,
```

Comment:
Q: shouldnt it be like this?

```suggestion
        tableName: pgSchema ? `"${pgSchema}"."${tableName}"` : tableName,
```

Reply:
Naah, the surrounding quotes are added by Mikro ORM already. If you look the output in the test

Reply:
yes indeed it is done by our util itself, not mikro orm. one question, I believe that the index name will be broken here no? should we instead handle the quotes in the util directly

Reply:
Yup. that makes sense. But let's do it in next PR.

Reply:
deal 👍